### PR TITLE
Added Learn Effection & Getting Started Guides

### PR DESCRIPTION
**Pull request by taras**
## Motivation

It's not clear right now how to install Effection.

## Approach

* Create an installation page. 
* Created a new section called "Learn Effection" similar to [React's documentation on react.dev ](https://react.dev/learn)
* Added "Add Effection to existing project"

## Screenshots

https://effection--tm-add-getting-started-gui.deno.dev

### CODE REVIEW

File: www/docs/installation.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## NPM/Yarn
+
+Effection packages are available via NPM and Yarn. The package includes ESM, CommonJS and TypeScript types.
+
+```bash
+// install with npm
+npm install effection@v3
+
+// install with yarn
+yarn add effection@v3
```
Comment (cowboyd) [inline www/docs/installation.mdx]:
```suggestion
// install with npm
npm install effection@next

// install with yarn
yarn add effection@next
```

Comment (cowboyd) [reply www/docs/installation.mdx]: Turns out `v3` is not a valid npm dist tag

File: www/docs/installation.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+Effection is designed to for gradual adoption. You can use as a little or as much Effection as you need.
+You can use Effection as a replacement for an async/await within your existing application, manage all of
+async operations in your application or anything in-between.
+
+Effection can be used in any JavaScript environment including Deno, Node and browsers. Effection packages are side-effect free and do not pollute your environment in any way, making it compatible with most JavaScript runtimes.
+If you encounter obstacles integrating with your environment, please create a [GitHub Issue in the Effection repository](https://github.com/thefrontside/effection/issues/new).
+
+Effection comes with [first-class TypeScript support](https://arethetypeswrong.github.io/?p=starfx%400.0.32), [is tree-shackable, has no dependencies and is small](https://bundlephobia.com/package/effection@3.0.0-beta.1).
```
Comment (cowboyd) [inline www/docs/installation.mdx]:
```suggestion
Effection comes with [first-class TypeScript support](https://arethetypeswrong.github.io/?p=effection@3.0.0-beta.1), [is tree-shackable, has no dependencies and is small](https://bundlephobia.com/package/effection@3.0.0-beta.1).
```

File: www/docs/installation.mdx (hunk 1 above)
Comment (taras) [reply www/docs/installation.mdx]: Yeah, I noticed that too

**Comment by cowboyd** [comment]
Instead of adding padding, what about letting the words wrap? That is what docusaurus seems to do....

**Pull request closed without merging**